tests/cpu-vm

```diff
@@ -15,11 +15,10 @@ if ! hasNeededAPIExtension cpu_hotplug; then
   exit 0
 fi
 
-# required for "CPU auto pinning" feature check
-# as we don't have a separate API extension for it
-# and we rely on the debug output in the LXD daemon logs.
-snap set lxd daemon.debug=true
-systemctl restart snap.lxd.daemon.service
+if ! hasNeededAPIExtension limits_cpu_pin_strategy; then
```

Comment:
I wouldn't go as far as skipping the whole test script. Could you rework it so that just the pinning checks depends on that API extension's presence?

Reply:
Sure thing!